src/Animals.java

```diff
@@ -0,0 +1,46 @@
+public class Animals {
+
+    private String name;
+    private static int animalsCount;
```

Comment:
Static variables should be declared first, then instance variables, then constructors, and the methods are the last members in the class order.